docs/tutorial_walk.ipynb

```diff
@@ -19,7 +19,8 @@
                 "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    engine._ensure_m_k_and_N_k_arrays(problem,m0)
</pre><p>why are we calling a private method here? make the method public or use a different method if this is not the intended workflow</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/symbiotic-engineering/OpenFLASH/pull/37/discussion/#comment-2463913542'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>